src/v2i-hub/ImmediateForwardPlugin/src/ImmediateForwardPlugin.h

```diff
@@ -26,61 +26,57 @@
 namespace ImmediateForward
 {
 
-struct MessageConfig
-{
-	uint ClientIndex;
-	std::string TmxType;
-	std::string SendType;
-	std::string Psid;
-	std::string Channel;
-};
+	struct MessageConfig
+	{
+		uint ClientIndex;
+		std::string TmxType;
+		std::string SendType;
+		std::string Psid;
+		std::string Channel;
+	};
 
 class ImmediateForwardPlugin : public tmx::utils::PluginClient
 {
-public:
-	ImmediateForwardPlugin(std::string name);
-	virtual ~ImmediateForwardPlugin();
-private:
-	void UpdateConfigSettings();
-	bool UpdateUdpClientFromConfigSettings(uint clientIndex);
-	bool ParseJsonMessageConfig(const std::string& json, uint clientIndex);
-	int GetUdpClientIndexForMessage(std::string subtype);
-	void SendMessageToRadio(IvpMessage *msg);
-	// @SONAR_STOP@
- 
-
-
-	// Virtual method overrides.
-	void OnConfigChanged(const char *key, const char *value);
-	void OnMessageReceived(IvpMessage *msg);
-	void OnStateChange(IvpPluginState state);
-
-
-	// Mutex along with the data it protects.
-	std::mutex _mutexUdpClient;
-	typedef std::vector<tmx::utils::UdpClient *> svr_list;
-	std::array<svr_list, 4> _udpClientList;
-	std::vector<MessageConfig> _messageConfigMap;
-	std::map<std::string, int> _messageCountMap;
-	std::string signatureData;
-	std::string url;
-	std::string baseurl;
-	std::string txMode;
-	unsigned int signState;
-	unsigned int enableHSM;
-
-	// Thread safe bool set to true the first time the configuration has been read.
-	std::atomic<bool> _configRead;
-
-	uint _skippedNoDsrcMetadata;
-	uint _skippedNoMessageRoute;
-	uint _skippedInvalidUdpClient;
-	uint _skippedSignErrorResponse;
-
-	bool _muteDsrc;
-	// @SONAR_START@
-
-};
+	public:
+		ImmediateForwardPlugin(std::string name);
+	private:
+		void UpdateConfigSettings();
+		bool UpdateUdpClientFromConfigSettings(uint clientIndex);
+		bool ParseJsonMessageConfig(const std::string& json, uint clientIndex);
+		int GetUdpClientIndexForMessage(std::string subtype);
+		void SendMessageToRadio(IvpMessage *msg);	
+
+
+		// Virtual method overrides.
+		void OnConfigChanged(const char *key, const char *value);
+		void OnMessageReceived(IvpMessage *msg);
+		void OnStateChange(IvpPluginState state);
+
+
+		// Mutex along with the data it protects.
+		std::mutex _mutexUdpClient;
+		using svr_list = std::vector<std::unique_ptr<tmx::utils::UdpClient>>;
```

Comment:
We could we a more descriptive name for this list if it is used in multiple places?